po/be.po

```diff
@@ -0,0 +1,937 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the org.gnome.Shell.Extensions.GSConnect package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: org.gnome.Shell.Extensions.GSConnect\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-12-19 02:37+0300\n"
+"PO-Revision-Date: 2018-12-19 13:33+0300\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"X-Generator: Poedit 2.2\n"
+"Last-Translator: Максім Крапіўка <metalomaniax@gmail.com>\n"
+"Language: be_BY\n"
```

Comment:
I just have one question before merging this. It's only because I'm clueless about `poedit` and about translations in general, so I'm being over-cautious.

I noticed that the language in `LINGUAS` and the filename is `be`, but the metadata generated by `poedit` shows `"Language: be_BY\n"` here. 

I checked, and **all** of the other translations list the same code in `Language:` that's used for the filename, so I'm wondering if it should be `"Language: be\n"` just to be safe? Or does it not matter?